src/components/DetailsView/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import ModiteContext from '../../state/modite';
+import Modite from '../../models/Modite';
+import classNames from 'classnames/bind';
+import s from './styles.module.css';
+import ModiteProfileResp from '../../models/ModiteProfileResp';
+
+// TODO: type correctly
+function Details({ className = '' } : any) {
+  const [activeModite, setActiveModite]: [Modite, React.Dispatch<any>] = useContext(ModiteContext);
+  const { profile = {} }: any = activeModite || {};
+  let { fields } = profile;
+
+  const fetchProfile = async () => {
+    if (!activeModite) return;
+
+    const moditeProfile: ModiteProfileResp = await fetch(
+      `https://modus.app/modite/${activeModite.id}`,
+    ).then(res => res.json());
+    activeModite.profile = moditeProfile.profile;
+    fields = moditeProfile.profile.fields;
+  }
+
+  if (!fields) {
+    fetchProfile();
+    fields = {};
+  }
+
+  const image = profile && profile.image_192;
+  const name = activeModite && activeModite.real_name ? activeModite && activeModite.real_name : activeModite && activeModite.name;
+  const { Location: location } = fields;
+  const tod = activeModite && activeModite.tod;
+  const localDate = activeModite && activeModite.localDate;
+  const localTime = activeModite && activeModite.localTime;
+
+  const cx = classNames.bind(s);
+  className = cx('moditeCt', className);
```

Comment:
Instead of `'moditeCt'`, could use `s.moditeCt` and that may catch if someone changes the CSS and not where it's used (for some human error reason) but I was playing around with this. I haven't had to use `classnames` lib much but in pass project I just did something like 

```
import cx from 'classnames';
import classes from './foo.css';
....
className={cx(classes.fooClass, classes.barClass)}
```